### ISSUE TITLE
When starting a game, starmap should be zoomed in to fit the view

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/open-ig#120]
```
Currently, it starts off as 540x380 or so. 

It should fit the screen.

Also, add CTRL+Middle click "Zoom to Fit" option.
```

Original issue reported on code.google.com by `akarn...@gmail.com` on 30 Oct 2011 at 10:19

**Comment by GoogleCodeExporter** [comment]
Original comment by `akarn...@gmail.com` on 30 Oct 2011 at 10:20
- Changed title: **When starting a game, starmap should be zoomed in to fit the view**

**Comment by GoogleCodeExporter** [comment]
```
Fixed: 0.93.475
```

Original comment by `akarn...@gmail.com` on 15 Dec 2011 at 10:37
- Changed state: **Fixed**
